src/BubbleSort.java

```diff
@@ -0,0 +1,20 @@
+import java.util.Arrays;
+
+public class BubbleSort {
+
+        public static void main(String[] args) {
+        // write your code here
+        int[] arr = {5, 2, 1, 10, 50, 10};
+        for (int N = 0; N < arr.length - 1; N++) {
```

Comment:
Please use letters `i` and `j` for the array iterations

Reply:
done